scripts/AbinsModules/Instruments/__init__.py

```diff
@@ -1,5 +1,13 @@
 # flake8: noqa
-from __future__ import (division, print_function)
-from ToscaInstrument import ToscaInstrument
-from TwoDMap import TwoDMap
-from Instrument import Instrument
+from __future__ import (absolute_import, division, print_function)
+# Make sure we can find Instruments...
```

Comment:
I must admit I am new to absolute_import, but @dymkowsk just checking if you don't include this work around will absolute_import not work?